aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,97 @@
+            ],
+            capture_output=True,
+            text=True
+        )
+        status = json.loads(result.stdout)["Invalidation"]["Status"]
+        if status == "Completed":
+            print(f"Invalidation {invalidation_id} completed")
+            break
+        print(f"Invalidation {invalidation_id} status: {status}")
+        time.sleep(10)  # Wait 10 seconds before checking again
+
+def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)